### ISSUE TITLE
:sparkles: Deprecate `Docker` images in favor of `Deno`'s task runner

### ISSUE THREAD

**Issue by x80486** [lumeland/docker#31]
While the `Docker` images could be convenient, with the insertion of `Deno`'s task runner, recent versions of `Lume` are able to use instead the [execute without install](https://lume.land/docs/overview/installation/#execute-it-without-install) option.

Basically, the `deno.json` configuration file could be used with `Lume` as well:

```javascript
{
  "tasks": {
    "build": "deno run -A https://deno.land/x/lume@v1.7.0/ci.ts",
    "serve": "deno task build -- --serve"
  }
}
```

In that sense, the version is always contained within the project, and all that's needed is to have a recent/compatible `Deno` version in the path.

**Comment by oscarotero** [comment]
I'm agree. [Deno tasks](https://deno.land/manual@v1.20.1/tools/task_runner) is pretty recent and yet unstable. I'm not sure when it will be stabilized, but I think after a couple of versions (deno 1.22) we could deprecate Docker image.

**Comment by x80486** [comment]
What do you think about generating these only for `Lume` versions `1.x`?

I was thinking that once `Lume` reaches `2.x`, every user should be aware of `Deno` tasks, and really the `Docker` images makes no sense at that point.

I will be maintaining these if you plan to maintain a `1.x` branch — or something around those lines.

**Comment by oscarotero** [comment]
Yes, I like it. I'm not sure when Lume will have a `2.0` version, maybe when Deno reaches to `2.0`.
